packages/node_modules/@webex/plugin-people/test/unit/spec/people-batcher.js

```diff
@@ -0,0 +1,97 @@
+/*!
+ * Copyright (c) 2015-2019 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {assert} from '@webex/test-helper-chai';
+import People from '@webex/plugin-people';
+import MockWebex from '@webex/test-helper-mock-webex';
+import sinon from 'sinon';
+
+describe('people-batcher', () => {
+  describe('PersonUUIDRequestBatcher', () => {
+    let batcher;
+    let webex;
+
+    beforeEach(() => {
+      webex = new MockWebex({
+        children: {
+          people: People
+        },
+        config: {
+          people: {showAllTypes: false}
+        }
+      });
+      batcher = webex.people.batcher;
+    });
+
+    // describe('#fingerprints', () => {
+    //   it('fingerprintRequest returns encrypted \'hydraId\'', () => batcher.fingerprintRequest('hydraId')
+    //     .then((result) => assert.deepEqual(result, webex.people.inferPersonIdFromUuid('hydraId'))));
+    // });
+
+    // describe('#submitHttpRequest()', () => {
+    //   const ids = ['id1'];
+    //   const mockRequest = {
+    //     service: 'hydra',
+    //     resource: `people/?id=${ids}&showAllTypes=false`
+    //   };
+
+    //   it('calls webex.request with expected params', () => {
+    //     webex.request = function (options) {
+    //       return Promise.resolve(options);
+    //     };
+
+    //     // webex.requestPromise = Promise.resolve(mockRequest);
+    //     return batcher.submitHttpRequest(ids)
+    //       .then((req) => assert.deepEqual(req, mockRequest));
+    //   });
+    // });
```

Comment:
New tests look good, curious about this section here. It seems as though this may have been missing coverage for this file. For the scope of this pull request, if these tests did not initially exist, we won't require this coverage for approval. But, adding additional coverage is welcome.

If you decide to not implement these tests, please remove the comment block.

Reply:
woops. that's not meant to be commented. there was no test coverage for people-batcher, so i added some :)